### ISSUE TITLE
Is support for s/nilable on the roadmap?

### ISSUE THREAD

**Issue by seancorfield** [wilkerlucio/spec-coerce#9]
I tried `(s/def ::maybe-int (s/nilable int?))` and `(sc/coerce ::maybe-int "42")` and got `"42"` back. I realized that I could use a long form spec with `s/or` but then I need to unconfirm the result.
```
(s/def ::maybe-int (s/or :nil nil? :int int?))
(sc/coerce ::maybe-int "42") => 42 ;; yay!
```

**Comment by wilkerlucio** [comment]
Good catch, we do the manual checking of the specs, so the `s/nilable` is probably not been taking into account, but we can make so it considers it. I'll try to work on that soon, thanks for the report.

**Comment by seancorfield** [comment]
👍

**Comment by wilkerlucio** [comment]
Just released on `1.0.0-alpha7`
